mediorum/server/monitor.go

```diff
@@ -109,20 +158,6 @@ func (ss *MediorumServer) updateDiskAndDbStatus() {
 	} else {
 		slog.Error("Error getting mediorum disk status", "err", err)
 	}
-
-	status := diskStatus{
-		StoragePathSizeGB: ss.storagePathSize / (1 << 30),
-		StoragePathUsedGB: ss.storagePathUsed / (1 << 30),
-		DatabaseSizeGB:    ss.databaseSize / (1 << 30),
-		Clock:             nearest5MinSinceEpoch(),
-	}
-	ss.logger.Info("updateDiskAndDbStatus", "diskStatus", status)
-}
-
-func nearest5MinSinceEpoch() time.Time {
-	secondsSinceEpoch := time.Now().Unix()
-	rounded := (secondsSinceEpoch + 150) / 300 * 300
-	return time.Unix(rounded, 0)
```

Comment:
i figured these should be removed when this merges. lmk if this was premature and i'll undo